project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.module

```diff
@@ -3350,6 +3403,36 @@ function c4m_message_format_node_share(Message $message) {
   );
 }
 
+/**
+ * Return the formatted unsubscribe url for taxonomy terms.
+ *
+ * @param Message $message
+ *   The Message entity.
+ *
+ * @return string
+ *   The unsubscribe link.
+ */
+function c4m_message_format_unsubscribe_topic_url(Message $message) {
+  return _c4m_message_format_unsubscribe_url($message, 'taxonomy_term');
+}
+
+/**
+ * Return the formatted share operation.
```

Comment:
Is the comment correct? Doesn't seem so.

Reply:
Fixed. At first we only needed notifications for topics. Generalised the naming now.